docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -256,17 +258,34 @@
               is also specified.</entry>
             </row>
 
+            <row>
+              <entry><emphasis role="bold">PLANE</emphasis></entry>
+
+              <entry>Specifies writing the <emphasis>indexfile</emphasis> to
+              the specified list of target planes. If omitted, the
+              <emphasis>indexfile</emphasis> is written to the default plane
+              for the cluster on which the workunit executes. The number of
+              physical file parts written to disk is always determined by the
+              number of nodes in the cluster on which the workunit executes,
+              regardless of the number of nodes on the target cluster(s)
+              unless the WIDTH option is also specified.</entry>
+            </row>
+
             <row>
               <entry><emphasis>target</emphasis></entry>
 
               <entry>A comma-delimited list of string constants containing the
-              names of the clusters to write the
-              <emphasis>indexfile</emphasis> to. The names must be listed as
-              they appear on the ECL Watch Activity page or returned by the
-              Std.System.Thorlib.Group() function, optionally with square
-              brackets containing a comma-delimited list of node-numbers
-              (1-based) and/or ranges (specified with a dash, as in n-m) to
-              indicate the specific set of nodes to write to.</entry>
+              names of the clusters or planes to write the
+              <emphasis>indexfile</emphasis> to. The cluster names must be
+              listed as they appear on the ECL Watch Activity page or returned
+              by the Std.System.Thorlib.Group() function, optionally with
```

Comment:
>The cluster names must be listed as they appear on the ECL Watch Activity page or returned by the Std.System.Thorlib.Group() function.

Not true in containerized. Planes are independent and there may be many (and I am not sure if they are listed in EclWatch anywhere at the moment - I've opened JIRA: https://hpccsystems.atlassian.net/browse/HPCC-32624).
And Thorlib.Group() makes no sense in containerized - there is no fixed group (of nodes) in cloud.